lib/jnpr/junos/device.py

```diff
@@ -412,6 +422,99 @@ def open(self, *vargs, **kvargs):
             originating ``Exception`` is assigned as ``err._orig``
             and re-raised to the caller.
         """
+        #
+        #  PRIVATE Functions
+        #
+
+        def _get_connected_slot():
+            """return connected slot. In case of error return None"""
+
+            rsp = self.rpc.get_chassis_inventory()
+            if rsp.tag == 'error':
+                raise RuntimeError('Could not determine RE slot id')
+            if rsp.tag == 'output':
```

Comment:
I can confirm this is an incorrect assumption for an MX VC. On an MXVC, where each member has dual REs, all 4 routing engines successfully execute the get-chassis-inventory RPC.